modules/liveIntentAnalyticsAdapter.js

```diff
@@ -1,134 +1,113 @@
 import {ajax} from '../src/ajax.js';
-import { generateUUID, logInfo, logWarn } from '../src/utils.js';
+import { generateUUID } from '../src/utils.js';
 import adapter from '../libraries/analyticsAdapter/AnalyticsAdapter.js';
 import { EVENTS } from '../src/constants.js';
 import adapterManager from '../src/adapterManager.js';
-import { auctionManager } from '../src/auctionManager.js';
 import { getRefererInfo } from '../src/refererDetection.js';
+import { config as prebidConfig} from '../src/config.js';
```

Comment:
```suggestion
import { config as prebidConfig } from '../src/config.js';
```
otherwise it fails the `lint` fails